### PR TITLE
Decimals in Barometer made consistent

### LINKED ISSUE #1633: Decimal degits in Default values in barometer not consistent
**Actual Behaviour**
Max, min, average pressure values have different numbers of decimal degits

**Expected Behaviour**
Number of decimal digits should be consistent

**Screenshots of the issue**
Screenshot_20190325-113941

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Smit2010**
Fixes #1633 
**Changes**: All the readings have been to rounded off to 2 decimal place.

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [app-debug.zip](https://github.com/fossasia/pslab-android/files/3055944/app-debug.zip)

### CODE REVIEW

**Comment by Smit2010** [comment]
@CloudyPadmal @abhinavraj23 @harsh-2711 Please review. Thanks.

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 1]
```diff
@@ -92,7 +92,7 @@
     public static final String LUXMETER_DATA_FORMAT = "%.2f";
     public static final String BAROMETER = "Barometer";
     public static final String BAROMETER_CONFIGURATIONS = "Barometer Configurations";
-    public static final String BAROMETER_DATA_FORMAT = "%.5f";
+    public static final String BAROMETER_DATA_FORMAT = "%.2f";
```
Comment (neel1998) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: @Smit2010  please confirm with some mentor regarding the precision required. Should it be .2f or .5f

Comment (Smit2010) [reply app/src/main/java/io/pslab/models/PSLabSensor.java]: Oh yes @neel1998. @CloudyPadmal could you please suggest the proper precision to be displayed here?
Thanks.

**Comment by Smit2010** [comment]
@CloudyPadmal @abhinavraj23 This issue can be merged, if no further changes is required. Thanks.

**Pull request merged**